src/parser/chordParserFactory.js

```diff
@@ -68,12 +69,42 @@ import formatSymbolParts from './filters/formatSymbolParts';
  * If multiple added/omits are present, the `add/omit` symbol is only printed once: `A+(add b9,#9)`
  */
 
+/**
+ * Intervals affected by the Alt modifier when parsing an altered chord written "C7alt", for example.
+ * @typedef {Object} AltIntervals
+ * @type {Object}
+ * @property {Boolean} fifthFlat - if the alt modifier should flatten the fifth
```

Comment:
dito

Reply:
yes, the API documentation is generated from the jsdoc ;)